package.json

```diff
@@ -62,7 +62,7 @@
     "happy-dom": "^9.20.3",
     "semantic-release": "^21.1.1",
     "shiki": "^0.14.3",
-    "unocss": "^0.55.3",
+    "unocss": "^0.56.0",
```

Comment:
I would maybe update unocss separately next time to avoid it breaking something in our packages. It is still v0 so minor updates could in theory be problematic for us :) It did happen when updating to v0.55 before. It resulted in `@warp-ds/uno` not working correctly, although all worked fine within component repos.